nextcloudappstore/core/templates/account/password_change.html

```diff
@@ -0,0 +1,47 @@
+{% extends "user/base.html" %}
+
+{% load css_class %}
+{% load i18n %}
+{% load account socialaccount %}
+
+{% block head-title %}{% trans "Sign In" %} - {% endblock %}
+
+{% block profile-content %}
+
+    <h1>{% trans "Change Password" %}</h1>
+
+    <form method="POST" action="{% url 'account_change_password' %}" class="password_change">
+        {% csrf_token %}
+        {% if form.non_field_errors  %}
+            {% for error in form.non_field_errors %}
+                <span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true"></span>
+                <span class="sr-only">(error)</span>
+                <p class="text-danger">{{ error|escape }}</p>
+            {% endfor %}
+        {% endif %}
+        {% for field in form %}
+        <div class="form-group {% if field.errors %}has-error has-feedback{% endif %}">
+            <label class="control-label" for="{{ field.id_for_label }}">{{ field.label }}</label>
+
+            {{ field|css_class:'form-control' }}
+
+            {% if field.errors %}
+                {% for error in field.errors %}
+                    <span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true"></span>
+                    <span class="sr-only">(error)</span>
+                    <p class="text-danger">{{ error|escape }}</p>
+                {% endfor %}
+            {% endif %}
+
+            {% if field.help_text %}
+                <p class="help-block">{{ field.help_text|safe }}</p>
```

Comment:
Ah right, it is :)

However I'd still remove safe, we dont want to add html to help_text